scripts/package/src/package_managers/brew.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+brew::install() {
+  # Some aliases
```

Comment:
Because `denisidoro/tools/docpars` it's not inside the main formulas yet.

Also this hasn't been touched, but since the file has been moved, it's highlighted as new ✨